### DISCUSSION TITLE
分配一部份 LUN 獎勵 TGE 結束前的貢獻

### DISCUSSION THREAD

**Discussion by yurenju** [lun-dao/LunDAO#26] [Ideas]
因為我們還沒開始放流動性，所以 LUN 代幣目前的價格也是未知，同時在這個時候其實大家已經開始貢獻專案了如參與討論、寫統計腳本跟架設靜態網頁產生器等。我想要先分配一些 LUN 來給 TGE 完成前的。

因為現在的規劃日常營運會有 200,000 LUN 可以使用，我想說我們先用約 5,000 LUN 分配給目前的貢獻者，然後或許試試看 [coordinape][1] 到底怎麼使用。理論上他是一個讓每個人思考一段時間內大家在社群裡面的合作對象，然後給他一個權重，最終會產生一個節點圖，照著這個節點圖分配獎勵。

可以參考以下影片看看 coordinape 到底是怎麼運作，也想知道大家有什麼其他的想法。

[![Coordinape 說明影片](http://img.youtube.com/vi/J8oGun8EKDE/0.jpg)](http://www.youtube.com/watch?v=J8oGun8EKDE)

[1]: https://coordinape.com/

**Comment by gasolin** [comment]
coordinape 看起來評價不錯 https://0xzx.com/zh-tw/2021102401041826978.html
可以先試看看
